clipboard/src/Clipboard.ts

```diff
@@ -11,7 +11,7 @@ export class Clipboard extends HTMLElement {
     const text = this.shadowRoot
       .querySelector('slot')
       .assignedNodes()[0]?.textContent;
-    copyTextToClipboard(text);
+    copyTextToClipboard.call(text);
```

Comment:
works for me without `.call`, with TS config or without
I think the issue was only with types, not with runtime behavior
it's a function in all cases